client/src/components/Loaders/PulseLoader.js

```diff
@@ -65,12 +65,21 @@ class PulseLoader extends PureComponent {
 }
 
 PulseLoader.propTypes = {
-  id: PropTypes.string.isRequired,
-  className: PropTypes.string.isRequired,
-  color: PropTypes.string.isRequired,
-  size: PropTypes.string.isRequired,
-  margin: PropTypes.string.isRequired,
-  verticalAlign: PropTypes.string.isRequired,
+  id: PropTypes.string,
+  className: PropTypes.string,
+  color: PropTypes.string,
+  size: PropTypes.string,
+  margin: PropTypes.string,
+  verticalAlign: PropTypes.string,
+};
+
+PulseLoader.defaultProps = {
```

Comment:
In the future, I have to review all the propTypes and to add defaultProps to the ones that are not required. Now it throws a lot of warnings and it doesn't look nice.